### ISSUE TITLE
Should the docs be significantly revised?

### ISSUE THREAD

**Issue by nickw1** [AR-js-org/AR.js-Docs#19]
Many of the docs are a bit out of date, particularly given the update to use Webpack and other signficant changes at 3.4.

Should we maybe significantly rewrite the docs and examples?

I'm happy, time permitting, to do this for location-based, though I don't have the expertise for the marker and NFT side of things.

**Comment by kalwalt** [comment]
Yes I think docs are a bit out of date too, if you can do this for location-based would be a nice thing.  I would do for the other (Marker, NFT) but now i start to be very busy with my job, so i can say when i could do this.

**Comment by nickw1** [comment]
@kalwalt no worries! I've been busy myself lately, hence only just dealing with the recent PRs ;-) I'll try and redo location-based as soon as I can.

**Comment by nickw1** [comment]
@kalwalt just one question: what's the difference between `ar.js`  and `ar-threex.js` ? One thing I can do is update the links to the marker-based and NFT builds on githack, but only if I understand the difference between these two. Thanks.

**Comment by kalwalt** [comment]
@nickw1 these two namespaces(?) or dist libs as you prefer, refer to two branch of the project `ar.js` that include `Context ` `Profile` `Session` classes and `new_api `necessary for the Aframe libs, new_api is indeed the wrapper for it. Note that `Context ` `Profile` are alias for ArToolkitContext, ArToolkitProfile, we only converted the code from the old implementation, but of course this not means that this is the best solution. The old AR.js code was created when classes designed in adifferent manner and modules (as we knows and use now) didn't exist.
We should think to rewrite maybe the code in a more modern and efficient fashion. That's was the idea of AR.js next.

**Comment by nickw1** [comment]
Partly done (for location-based) by PR #20.

**Comment by AzimovS** [comment]
Thank you very much for the all hard work! Here is very minor fix that I found #22

**Comment by nickw1** [comment]
@AzimovS Thanks! Have now merged your PR. I also need to apply a further fix (see PR for details) so that the accompanying text is accurate.
